src/components/Donation/utils/stripe.ts

```diff
@@ -1,4 +1,4 @@
-import { DonationMachineContext } from './machine/types';
+import { DonationMachineContext } from '../machines/donationType';
```

Comment:
are we sure about this? There are couple subtle differences between typing of the two machines, that's why we have tickets such as https://app.asana.com/0/1195530778765030/1195530778765039

Reply:
There was an eslint error message regarding this import

```
Cannot find module './machine/types' or its corresponding type declarations.ts(2307)
```

Since there is no folder named `machine` I replaced it with `machines`

The only place where we have a `DonationMachineContext` declaration it's in `src/components/Donation/machines/donationType.ts` line 74

```ts
export type DonationMachineContext = {
  /**
   * A key to hold the response once the donation service respond
   */
  donation: {
    message: string;
    status: string;
  };
```

Reply:
I wonder how this happen all the way to master without us to realising it, that's why I am a bit confused. Furthermore, I don't get errors on my local. This seems to be odd.

_But I guess if you double check and everything points to this change then let's go!_

Reply:
Seems we don't have a CI for this project in order to avoid this things to happen.
https://app.asana.com/0/1195530778765030/1199125464121213